Sources/OpenAI/Private/URLSessionDataTaskProtocol.swift

```diff
@@ -11,6 +11,7 @@ import FoundationNetworking
 #endif
 
 protocol URLSessionTaskProtocol: Sendable {
+    var originalRequest: URLRequest? { get }
```

Comment:
Is it being provided/set anywhere?

Reply:
This is a default property from `URLSessionTask`, so no need to provide it separately - it's provided by Foundation
<img width="649" alt="Screenshot 2025-03-24 at 21 39 58" src="https://github.com/user-attachments/assets/49533e3b-83d9-44bc-b6a0-96980a51536c" />

Reply:
Ok I see, thanks!